scripts/src/index.js

```diff
@@ -34,27 +33,29 @@ const r = new snoowrap({
 
 (async () => {
 	try {
-    const txconfig = {
-      watcher: watcher,
-      burnAddr: burnAddr,
-      curr: curr,
-      limit: 100
-    }
-    const burned = await tx.getAllBurned(txconfig)
-    const getFlair = flair.flairGetter(txconfig, burned, util.validPurchase)
-    const rUsers = await flair.getUserMap(userAddressUrl)
-    const flairArrays = flair.buildMultipleUserFlairs(
-      rUsers,
-      getFlair('flair_rock', ':rock:', price),
-      getFlair('flair_salamander', ':salamander:', 7000),
-      getFlair('flair_soon', ':soon:', 10000)
-    )
-    const update = await flair.setFlairs(
-      flairArrays,
-      r.getSubreddit(subreddit)
-    )
-    console.log(update)
-  } catch(err) {
+    cron.schedule('*/30 * * * *', async () => {
+      const txconfig = {
+        watcher: watcher,
+        burnAddr: burnAddr,
+        curr: curr,
+        limit: 200
+      }
+      const burned = await tx.getAllBurned(txconfig)
+      const getFlair = flair.flairGetter(txconfig, burned, util.validPurchase)
+      const rUsers = await flair.getUserMap(userAddressUrl)
+      const flairArrays = flair.buildMultipleUserFlairs(
+        rUsers,
+        getFlair('flair_rock', ':rock:', 1500),
+        getFlair('flair_salamander', ':salamander:', 7000),
+        getFlair('flair_soon', ':soon:', 10000)
+      )
+      console.log(flairArrays)
+      const update = await flair.setFlairs(
+        flairArrays,
+        r.getSubreddit(subreddit)
+      )
+      console.log(update)
+    })} catch(err) {
     console.log(err)
```

Comment:
should we log to sentry? potentially noisy but maybe better than waiting for a user to report support

Reply:
yup, I'm planning to

Reply:
+1 @nicholasmueller